src/WFS.js

```diff
@@ -163,14 +169,127 @@ L.WFS = L.FeatureGroup.extend({
 
         return that;
       },
-      error: function(errorMessage) {
+      error: function (errorMessage) {
         that.fire('error', {
           error: new Error(errorMessage)
         });
 
         return that;
       }
     });
+  },
+
+  getCapabilities: function (successCallback, errorCallback) {
+    var capabilities = this._capabilities;
+
+    // Check if capabilities were already received & cached.
+    if (capabilities) {
+      if (typeof (successCallback) === 'function') {
+        successCallback(capabilities);
+
+        return;
+      }
+    }
+
+    var requestData = L.XmlUtil.createElementNS('wfs:GetCapabilities', {
+      service: 'WFS',
+      version: this.options.version
+    });
+
+    var that = this;
+    L.Util.request({
+      url: this.options.url,
+      data: L.XmlUtil.serializeXmlDocumentString(requestData),
+      headers: this.options.headers || {},
+      success: function (data) {
+        // If some exception occur, WFS-service can response successfully, but with ExceptionReport,
+        // and such situation must be handled.
+        var exceptionReport = L.XmlUtil.parseOwsExceptionReport(data);
+        if (exceptionReport) {
+          if (typeof (errorCallback) === 'function') {
+            errorCallback(exceptionReport.message);
+          }
+
+          return;
+        }
+
+        // Request was truly successful (without exception report), parse WFS_Capabilities.
+        capabilities = L.XmlUtil.parseXml(data).documentElement;
+
+        // Cache received capabilities.
+        that._capabilities = capabilities;
+
+        if (typeof (successCallback) === 'function') {
+          successCallback(capabilities);
+        }
+      },
+      error: function (errorMessage) {
+        if (typeof (errorCallback) === 'function') {
+          errorCallback(errorMessage);
+        }
+      }
+    });
+  },
+
+  getBoundingBox: function (successCallback, errorCallback) {
```

Comment:
Unit tests for 'getBoundingBox' are necessary. To save time you could get them from Leaflet-WMS plugin and adapt them to this plugin.